packages/grafana-toolkit/docker/grafana-plugin-ci/build.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
+set -eo pipefail
+
 source ./common.sh
 
-output=$(docker build . | tee /dev/tty)
-hash=$(echo "$output" | tail -1 | sed -ne "s/^Successfully built \(.*\)/\1/p")
-docker tag "$hash" $DOCKER_IMAGE_NAME:latest
-docker push $DOCKER_IMAGE_NAME:latest
+docker build -t ${DOCKER_IMAGE_NAME} .
```

Comment:
Just to make sure, there's no need to tag the image any more?

Reply:
@wbrowne It still gets tagged, as part of $DOCKER_IMAGE_NAME. It ensures consistency, since the tag was otherwise repeated.

Reply:
Ah interesting - didn't know you can collapse into `build`!

Reply:
@wbrowne Oh yeah, I didn't quite understand your exact question until now :D